src/index.ts

```diff
@@ -76,7 +76,7 @@ export async function argosScreenshot(
   await page.addStyleTag({ content: GLOBAL_STYLES });
 
   // Wait for all busy elements to be loaded
-  await page.waitForSelector('[aria-busy="true"]', { state: "hidden" });
+  await page.waitForSelector('[aria-busy="true"]', { state: "detached" });
```

Comment:
This is not what we want here. A "display: none" aria-busy does not count so "state: hidden" is the good one.

Reply:
please read my comment

Reply:
Ok, I think this fix is not good. It will break things. But did we have some 0x0px aria-busy in Pivot? I think not.